src/pytorch_lightning/utilities/warnings.py

```diff
@@ -39,6 +40,11 @@ def deprecation(self, message: str, stacklevel: int = 5, **kwargs: Any) -> None:
             self.add(message)
             new_rank_zero_deprecation(message, stacklevel=stacklevel, **kwargs)
 
+    def info(self, message: str, stacklevel: int = 5, **kwargs: Any) -> None:
```

Comment:
we can rename WarningCache to `LoggingCache` if required since `info` is not a warning.